site/_posts/2019-10-25-contour-v100rc2.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Contour 1.0.0-rc.2 has been released
+excerpt: Contour 1.0 is just around the corner.
+author_name: Dave Cheney
+author_avatar: /img/contributors/dave-cheney.png
+categories: [kubernetes]
+# Tag should match author to drive author pages
+tags: ['Contour Team', 'release']
+---
+
+Contour 1.0.0-rc.2 is the second, and hopefully final, release candidate on the path to Contour 1.0.
+Assuming that no serious issues are found next week we're on track to release Contour 1.0 on November 1st.
+
+It goes without saying that without the help of the many community contributors this release, nor the 38 that preceeded it, would not have been possible.
+Thank you all.
+
+You can read the full [1.0.0-rc.2 release notes](https://github.com/projectcontour/contour/releases/tag/v1.0.0-rc.2) over on GitHub, but as you're here, here are a few highlights. 
+
+## Website improvements
+
+As part of the continued preparations for the 1.0 release Contour's documentation has been relocated to the projectcontour.io website. Specifically;
+
+* The Getting Started documentation has moved to [projectcontour.io/getting-started]({% link getting-started.md %})
+* Guides and How-to's have moved to [projectcontour.io/guides]({% link guides.md %})
+* Versioned release documentation has moved to [projectcontour.io/docs]({% link docs.md %})
+* Project related and non-versioned documentation has moved to [projectcontour.io/resources]({% link resources.md %})  
+
+We're working hard to polish the website content ahead of the 1.0 release. Please pardon our dust.
+
+## IngressRoute and HTTPProxy status update improvements
+
+IngressRoute and HTTPProxy status updates are now performed by the lead Contour in the deployment.
+We're hopeful that this will put to reset the many issues related to status update loops over the years.
+
+## HTTPProxy and IngressRoute OpenAPIv3 schema validation
+
+Contour 1.0.0-rc.2 includes updated OpenAPIv3 schema validations.
+These schemas are automatically generated from the CRDs themselves and should be more complete and consistent than the previous hand rolled versions.
+
+## TCPProxy delegation
+
+Contour 1.0.0-rc.2 now supports TCPProxy delegation. See the [relevant section]({% link _docs_1_0/httpproxy.md %}) in the HTTPProxy documentation.
+
+## Envoy keepalive tuning
+
+
+A recurrent issue when deploying Contour on clusters where overlay networking was used was long running connections between Contour and Envoy could become stuck half-open (one side thinks the connection is open, the other side doesn't) or half-closed (one side closes the connection, the other side never gets the message).
```

Comment:
Rewrite to:
"A recurrent issue when deploying Contour on clusters and using overlay networking, was that long running connections between Contour and Envoy could become stuck half-open (one side thinks the connection is open, the other side doesn't) or half-closed (one side closes the connection, the other side never gets the message)."